content/reference/endpoints/inbox_participants.md

```diff
@@ -0,0 +1,85 @@
+# Inbox Participants
+
+1. TOC
+{:toc}
+
+### Parameters
+<ul class="nav nav-pills" role="tablist">
+  <li class="disabled"><a>OAuth Scopes:</a></li>
+  <li class="active"><a href="#inbox_read" role="tab" data-toggle="pill">inbox_read</a></li>
+  <li><a href="#inbox_write" role="tab" data-toggle="pill">inbox_write</a></li>
+</ul>
+<div class="tab-content" markdown="1">
+  <div class="tab-pane active" id="inbox_read" markdown="1">
+Name                 | Type    | Read/Write | Description
+---------------------|---------|------------|------------
+id                   | Integer | Read       | Participant's id.
+---------------------|---------|------------|------------
+created_at           | [Time](/reference/enums#formats) | Read       | Participant's create time.
+updated_at           | [Time](/reference/enums#formats) | Read       | Participant's update time.
+last_message_sent_at | [Time](/reference/enums#formats) | Read       | Participant's last sent message time.
+read_at              | [Time](/reference/enums#formats) | Read       | Time when Participant read last message.
+{: class="table table-bordered"}
+  </div>
+  <div class="tab-pane" id="inbox_write" markdown="1">
+Name                 | Type    | Read/Write | Description
+---------------------|---------|------------|------------
+id                   | Integer | Read       | Participant's id.
+member_id            | Integer | Read/Write | Participant's Member id.
+conversation_id      | Integer | Read/Write | Participant's Conversation id.
+---------------------|---------|------------|------------
+read                 | Boolean | Read/Write | Is related to Participant's read_at attribute.
```

Comment:
`read` is write only, `read_at` is both read/write, but `read` is really the same as sending read_at with Time.current